puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/smart-proxy-plugin.yaml

```diff
@@ -0,0 +1,18 @@
+- builder:
+    name: smart-proxy-plugin
+    builders:
+      - shell: |+
+          #!/bin/bash -ex
+
+          # RVM Ruby environment
+          . /etc/profile.d/rvm.sh
+
+          # Use a gemset unique to each executor to enable parallel builds
+          gemset=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
+          rvm use ruby-${ruby}@${gemset} --create
+          rvm gemset empty --force
+          gem install bundler --no-ri --no-rdoc
+
+          bundle install --retry 5
```

Comment:
Could you apply tricks from https://github.com/theforeman/foreman-infra/pull/628 here as well? Hiding RVM install and multiple jobs should make for a better user experience for the dev.

Reply:
tricksies increased

![tricksy gollum](https://i.pinimg.com/originals/29/c6/56/29c6561ab8341623e6c8ed438a475bdb.jpg)